### ISSUE TITLE
FTI plugin building fail

### ISSUE THREAD

**Issue by jbigot** [pdidev/pdi#72]
In GitLab by @ksiero on Mar 9, 2018, 13:12

On my and Tomek's systems we can't build FTI plugin. The problem is in the `pdi/plugins/fti/fti.cxx` file the `#include <iostream>` is missing.

Log:
```
canning dependencies of target pdi_fti_plugin
[ 97%] Building CXX object plugins/fti/CMakeFiles/pdi_fti_plugin.dir/fti.cxx.o
/home/sierot/Workspace/pdi/plugins/fti/fti.cxx:57:12: error: ‘std::cerr’ has not been declared
 using std::cerr;
            ^
/home/sierot/Workspace/pdi/plugins/fti/fti.cxx:58:12: error: ‘std::endl’ has not been declared
 using std::endl;
            ^
```

**Comment by jbigot** [comment]
you can create a branch to add the missing include

**Comment by jbigot** [comment]
created branch [`72-fti-plugin-building-fail`](https://gitlab.maisondelasimulation.fr/jbigot/pdi/compare/master...72-fti-plugin-building-fail)

**Comment by jbigot** [comment]
mentioned in commit 5c507944ac15a58dc32da80bbd33f1b9927e6e4c

**Comment by jbigot** [comment]
closed via merge request !103

**Comment by jbigot** [comment]
closed via commit 5c507944ac15a58dc32da80bbd33f1b9927e6e4c
